### PR TITLE
export apns:connect/1

### PR DESCRIPTION
**Pull request by dcy**
1. export ```apns:connect/1```, so i can handle two production apps(open two connections) in one Erlang application.

1. add ```jsx``` to ```apns.app.src```, i don't need to add ```jsx``` to my app's(top level app) xxx.app.src

1. Readme add ```apns:connect/1```

### CODE REVIEW

File: src/apns.app.src [hunk 1]
```diff
@@ -6,6 +6,7 @@
   {applications, [
     kernel,
     stdlib,
+    jsx,
```
Comment (ferigis) [inline src/apns.app.src]: We don't need to add jsx here. jsx is a library, not an application, so we don't have to start anything.

**Comment by ferigis** [comment]
Hi @dcy , thanks for the PR :)
I have just requested a little change. After that I will merge it

File: src/apns.app.src (hunk 1 above)
Comment (elbrujohalcon) [inline src/apns.app.src]: Actually, you probably need to add it to ease rebar3 release support.

Comment (ferigis) [inline src/apns.app.src]: approved then!

**Comment by dcy** [comment]
By the way. Do you want to change jsx to [jsone](https://github.com/sile/jsone#benchmark)? I can make another PR.

**Comment by elbrujohalcon** [comment]
Not really, most of our libraries use jsx. What are the benefits of jsone?

**Comment by dcy** [comment]
According to the [benchmark](https://github.com/sile/jsone#benchmark), jsone have better performance.

**Comment by elbrujohalcon** [comment]
Yeah, I don't feel compelled to change it just because of that. What do you think, @ferigis ?

**Comment by ferigis** [comment]
I think we are fine with jsx. We don't have performance issues and we are align with our other projects. If we experience some performance problems in future we will take jsone into account for sure. Thanks for the idea

**Pull request merged**